mono_repo/mono_pkg.yaml

```diff
@@ -8,6 +8,8 @@ stages:
   - group:
     - dartfmt
     - dartanalyzer: --fatal-infos --fatal-warnings .
+    # A bit weird, this should really be a global task but we don't have those.
```

Comment:
How about making this a global task by default?

Reply:
I started there but it was ugly and we would need other refactoring - tasks require an `os`, `sdk`, etc which would need to be configurable.

I think we do want to add the general ability to configure global tasks but I don't think we can do a sane default here.

Reply:
Maybe link to the issue in this comment block?